src/routers.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {
+    BrowserRouter as Router,
+    Route,
+    Switch
+} from 'react-router-dom'
+import App from './App';
+import login from './views/login';
+
+const Routes = () =>(
+    <Router>
+        <h1>Anything that has to appear on all pages</h1>
```

Comment:
remove this h1. It is better to have only the routes in this finction

Reply:
done